package.json

```diff
@@ -29,7 +29,8 @@
     "mustache": "0.8.2",
     "node-uuid": "1.4.1",
     "logops": "0.2.0",
-    "node-cache": "1.0.3"
+    "node-cache": "1.0.3",
+    "underscore": "*"
```

Comment:
Maybe a particular version should be enforced, in order to have a "frozen" view of the dependencies (reduces ambiguity).